common/src/main/java/com/tencent/rss/common/config/RssConf.java

```diff
@@ -576,7 +576,7 @@ public <T> Optional<T> getOptional(ConfigOption<T> option) {
     return value;
   }
```

Comment:
I suggest you that you read Flink configuration implement carefully. You should also modify here.

```  
public <T> Optional<T> getOptional(ConfigOption<T> option) {
        Optional<Object> rawValue = getRawValueFromOption(option);
        Class<?> clazz = option.getClazz();

        try {
            if (option.isList()) {
                return rawValue.map(v -> ConfigurationUtils.convertToList(v, clazz));
            } else {
                return rawValue.map(v -> ConfigurationUtils.convertValue(v, clazz));
            }
        } catch (Exception e) {
            throw new IllegalArgumentException(
                    String.format(
                            "Could not parse value '%s' for key '%s'.",
                            rawValue.map(Object::toString).orElse(""), option.key()),
                    e);
        }
```

Reply:
Although the RssConf impl looks like the Flink conf, but the convert method is different. That means we shouldn't do the convert action in `getOptional` method, just to init it in `TypedConfigOptionBuilder` or `ListConfigOptionBuilder`.

Now in current PR, i dont implement the case like `conf.set(listStringOption, List("a", "b", c))`. I think i can support it in the `ListConfigOptionBuilder` directly. No need to change the `getOptional` method.

Reply:
And we should allow users to set conf kv in the following usages.
1. conf.set("rss.key", "a,b,c");
2. conf.set(ConfigOption<T> key, T value);

Reply:
````
  public <T> Optional<T> getOptional(ConfigOption<T> option) {
    Optional<Object> rawValue = getRawValueFromOption(option);
    Class<?> clazz = option.getClazz();
    Optional<T> value = rawValue.map(v -> option.convertValue(v, clazz));
    return value;
  }
```
This is the RssConf's implement. Why do we implement the case like conf.set(listStringOption, List("a", "b", c))? 
It's not consistent with others.

Reply:
This case has been supported in latest commit.

Reply:
Flink Configuration's implement seems more complete. Method `StructuredOptionsSplitter` support more split characters. Should we reuse Flink's implement?

Reply:
If you reuse the Flink's logic, it will also be easy to be reviewed, thanks.

Reply:
I think there is no need to reuse it. Just split it in ConfigOption is enough.